File: cpp/gabaritos/gabarito4.cpp
#include <iostream>

int main() {
    int n;
    std::cout << "Digite um número: ";
    std::cin >> n;
    int fatorial = 1;
    for (int i = 1; i <= n; i++) {
        fatorial *= i;
    }
    std::cout << "O fatorial de " << n << " é " << fatorial << std::endl;
    return 0;
}

File: csharp/gabaritos/gabarito4.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite um número: ");
        int n = int.Parse(Console.ReadLine());
        int fatorial = 1;
        for (int i = 1; i <= n; i++)
        {
            fatorial *= i;
        }
        Console.WriteLine($"O fatorial de {n} é {fatorial}");
    }
}